### PR TITLE
Adding Pundit Authorization

### PR DESCRIPTION
**Pull request by mlemansk**
In this pull request, I added Pundit policies and authorization to my application.  I would like some feedback if I followed typical rails convention in creating these policies.  One place I wasn't sure on is what the authorization should look like in the `new?` and `create?` policies, since there is not a record yet.  Since as of now, any user can create a new record, I defaulted those both to `true`.  Thanks for reviewing!

### CODE REVIEW

File: app/controllers/adjustments_controller.rb [hunk 1]
```diff
@@ -1,26 +1,33 @@
...
   end
 
   # GET /adjustments/1/edit
   def edit
+    authorize @adjustment
   end
 
   # POST /adjustments or /adjustments.json
   def create
+    authorize Adjustment
```
Comment (jelaniwoods) [inline app/controllers/adjustments_controller.rb]: You can authorize the new record should have all the initial columns set, except for `id`, `created_at`, `updated_at`. So if you needed to restrict based on user or role in the policy you could.
```suggestion
    @adjustment = Adjustment.new(adjustment_params)
    authorize @adjustment
```

File: app/controllers/adjustments_controller.rb [hunk 2]
```diff
@@ -1,26 +1,33 @@
...
   end
 
   # GET /adjustments/1 or /adjustments/1.json
   def show
+    authorize @adjustment
   end
 
   # GET /adjustments/new
   def new
+    authorize Adjustment
```
Comment (jelaniwoods) [inline app/controllers/adjustments_controller.rb]: If you are going to keep the `new` action as `true` you can still pass the new record to the `authorize` method.

```suggestion
    @adjustment = Adjustment.new
    authorize @adjustment
```

**Review by jelaniwoods — COMMENTED** [review]
@mlemansk nice 👍🏽 I left some comments about policies for `new` and `create`. Let me know what you think.

File: app/controllers/adjustments_controller.rb (hunk 1 above)
Comment (mlemansk) [reply app/controllers/adjustments_controller.rb]: This makes sense.  I will add this to both the `loan` and `adjustment` controller. Thanks!

File: app/controllers/adjustments_controller.rb (hunk 2 above)
Comment (mlemansk) [reply app/controllers/adjustments_controller.rb]: This makes sense.  I will add this to both the `loan` and `adjustment` controller. Thanks!

**Pull request closed without merging**